### ISSUE TITLE
Binding Cmd + W to close window button.

### ISSUE THREAD

**Issue by fermata** [OrangeDrangon/android-messages-desktop#377]
**Is your feature request related to a problem? Please describe.**
I cannot close the window with <kbd>Cmd</kbd> + <kbd>W</kbd> shortcut like the other apps do.

**Describe the solution you'd like**
Maybe closing a window?

Thanks for the great app by the way.

**Comment by OrangeDrangon** [comment]
I do not have a mac to test but CTRL+W works on linux. I assume there is a platform related issue going on.

**Comment by innesst** [comment]
CTRL+W closes on windows 10

**Comment by OrangeDrangon** [comment]
https://github.com/OrangeDrangon/android-messages-desktop/blob/ff20c5141fc111730580046682860374d6f14f2f/src/menu/windowMenu.ts#L3

This is the code for the window menu. It is just relying on an electron provided menu template.

**Comment by anonymousaga** [comment]
+1

**Comment by OrangeDrangon** [comment]
according to some research cmd+q is the global window close on macos. closing wont fix

**Comment by anonymousaga** [comment]
I disagree with this. Cmd+q is quit. Cmd+w is to close all windows. This app can run without any windows (notifications), so it makes sense to have cmd+w & cmd+q.

**Comment by OrangeDrangon** [comment]
fair enough this can stay open but the change will need to come from someone else.

**Comment by mananong** [comment]
+1 on this feature please, amazing app nonetheless thank you!!

**Comment by ilovepancakes95** [comment]
+1 on this

**Comment by anonymousaga** [comment]
I have submitted a pull request to add cmd+w to this app. 

@OrangeDrangon please check it out. Thanks

https://github.com/OrangeDrangon/android-messages-desktop/pull/459
